src/canvas-view/canvas/canvas-component.js

```diff
@@ -0,0 +1,393 @@
+(function () {
+  'use strict';
+
+  angular.module('patternfly.canvas')
+    .filter('trustAsResourceUrl', ['$sce', function ($sce) {
+      return function (val) {
+        return $sce.trustAsResourceUrl(val);
+      };
+    }])
+
+    //
+    // Component that generates the rendered chart from the data model.
+    //
+    .component('pfCanvas', {
+      templateUrl: 'canvas-view/canvas/canvas.html',
+      bindings: {
+        chartDataModel:'=',
+        chartViewModel: '=?',
+        readOnly: '=?',
```

Comment:
This is non-breaking, but we use a one way bind on the other readOnly binding would that work here too?

Reply:
Yes it does work here, will change for consistency.